docs/source/references/schema.json

```diff
@@ -3739,6 +4016,109 @@
       }
     ]
   },
+  "SingularQueryBox": {
+    "Enum": [
+      {
+        "tag": "FindAssetQuantityById",
+        "discriminant": 0,
+        "type": "FindAssetQuantityById"
+      },
+      {
+        "tag": "FindExecutorDataModel",
+        "discriminant": 1,
+        "type": "FindExecutorDataModel"
+      },
+      {
+        "tag": "FindAllParameters",
```

Comment:
This name stands out within singular queries and is confusing. `FindParameters`? `FindConfiguration`? @mversic

Reply:
yes, we should remove `All` from query names, I commented on that. also remember to update `visit` method name

Reply:
Whoops, I forgot to update this enum variant.

And also the metric names... Will fix it